packages/api/src/patchStyleOptions.js

```diff
@@ -32,95 +14,7 @@ export default function patchStyleOptions(
   // Keep this list flat (no nested style) and serializable (no functions)
 
   // TODO: [P4] Deprecate this code after bump to v5
```

Comment:
remove

Reply:
This is for v5 notes ^^, not the code I deleted